Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -307,6 +307,31 @@ def get_tenant_info(self):
             timeout=self.timeout
         )
         return reply.get('reply', {})
+    
+    def get_multiple_incidents_extra_data(self, incident_id):
```

Comment:
I dont get this method, we use an endpoint called get multiple incidents extra data, does it support more than one incident? also, the method's name is misleading, if its for one incident, lets change it.